docs/release_notes/release-3.0.md

```diff
@@ -1,5 +1,39 @@
+- Supports the following [window functions](../sql-reference/sql-functions/Window_function.md): COVAR_SAMP, COVAR_POP, CORR, VARIANCE, VAR_SAMP, STD, and STDDEV_SAMP.
+
+## Improvements
+
+- Added more prompts in the error message `xxx too many versions xxx`. [#28397](https://github.com/StarRocks/starrocks/pull/28397)
+- Dynamic partitioning further supports the partitioning unit to be year. [#28386](https://github.com/StarRocks/starrocks/pull/28386)
+- The partitioning field is case-insensitive when expression partitioning is used at table creation and [INSERT OVERWRITE is used to overwrite data in a specific partition](../table_design/expression_partitioning.md#load-data-into-partitions). [#28309](https://github.com/StarRocks/starrocks/pull/28309)
+
+## Bug Fixes
+
+Fixed the following issues:
+
+- Incorrect table-level scan statistics in FE cause inaccurate metrics for table queries and loading. [#27779](https://github.com/StarRocks/starrocks/pull/27779)
+- The query result is not stable if the sort key is modified for a partitioned table. [#27850](https://github.com/StarRocks/starrocks/pull/27850)
+- The version number for a tablet is inconsistent between the BE and FE after data is restored. [#26518](https://github.com/StarRocks/starrocks/pull/26518/files)
+- The number of buckets is automatically inferred as 0 if the number of buckets is not specified at the creation of a Colocation table, which causes failure of adding new partitions. [#27086](https://github.com/StarRocks/starrocks/pull/27086)
+- When the SELECT result set of INSERT INTO SELECT is empty, the output of SHOW LOAD shows the load job status is `CANCELED`. [#26913](https://github.com/StarRocks/starrocks/pull/26913)
+- BEs may crash when the input values of the sub_bitmap function are not of the BITMAP type. [#27982](https://github.com/StarRocks/starrocks/pull/27982)
+- BEs may crash when the AUTO_INCREMENT column is being updated. [#27199](https://github.com/StarRocks/starrocks/pull/27199)
+- Outer join and Anti join rewrite errors for materialized views. [#28028](https://github.com/StarRocks/starrocks/pull/28028)
+- Inaccurate estimation of average row size causes Primary Key partial updates to occupy excessively large memory. [#27485](https://github.com/StarRocks/starrocks/pull/27485)
+- Activating an inactive materialized view may cause a FE to crash. [#27959](https://github.com/StarRocks/starrocks/pull/27959)
+- Queries can not be rewritten to materialized views created based on external tables in the Hudi catalog. [#28023](https://github.com/StarRocks/starrocks/pull/28023)
+- You can still query the data of the Hive table even if you drop the Hive table and manually update the metadata cache. [#28223](https://github.com/StarRocks/starrocks/pull/28223)
```

Comment:
suggest: The data of a Hive table can still be queried even after the table is dropped and the metadata cache is manually updated.